scripts/install_dev_server.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
```

Comment:
I want to avoid adding another very similar script that we have to maintain. What's the actual difference between this and install_from_src? Can we just add another flag/envvar to that to cover what you're doing?